api/src/search/backend/load_opportunities_to_index.py

```diff
@@ -269,6 +275,31 @@ def fetch_existing_opportunity_ids_in_index(self) -> set[int]:
 
         return opportunity_ids
 
+    def filter_attachments(
+        self, attachments: List[OpportunityAttachment], filters: list
+    ) -> List[OpportunityAttachment]:
+        return [attachment for attachment in attachments if attachment.mime_type in filters]
```

Comment:
The filters will be a bit different than this, I'd just make it do `return [attachment for attachment in attachments]` for now